components/drops/useDrops.ts

```diff
@@ -53,3 +59,31 @@ export async function useDrop(id: string) {
 
   return drop
 }
+
+export const useDropStatus = (id: string) => {
+  const currentAccountMintedToken = ref<{
+    created_at: string
+    id: number
+    image: string
+    metadata: string
+  } | null>(null)
+  const mintedDropCount = ref(0)
+  const { accountId } = useAuth()
+
+  const fetchDropStatus = async () => {
+    const { count } = await getDropStatus(id)
+    mintedDropCount.value = count
+    currentAccountMintedToken.value = accountId.value
+      ? await getDropMintedStatus(id, accountId.value)
```

Comment:
types from `getDropMintedStatus()` and `currentAccountMintedToken` do not match

Reply:
updated